use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: default
+title: "Beam WordCount Example"
+permalink: /use/wordcount-example/
+redirect_from: /docs/use/wordcount-example/
```

Comment:
Can ditch this. (Is only needed on pages that got moved as part of the site reorg a couple of months ago.)